languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -1049,7 +1049,52 @@ though, and has since been slightly modified)-->
     </rule>
   </category>
   <category id="CAT_HOMONYMES_PARONYMES" name="Confusion d’homonymes et paronymes">
-    <rule id="GENTE_GENT" name="gente → gent">
+      <rulegroup id="PART_OU_PAR" name="part ou par" default="temp_off">
+          <rule>
+              <antipattern>
+                  <token>part</token>
+                  <token postag="P.*" postag_regexp="yes"/>
+              </antipattern>
+              <antipattern>
+                  <token>de</token>
+                  <token>part</token>
+                  <token>et</token>
+                  <token min="0">d'</token>
+                  <token>autre</token>
+              </antipattern>
+              <antipattern>
+                  <token>de</token>
+                  <token>part</token>
+                  <token>en</token>
+                  <token>part</token>
+              </antipattern>
+              <pattern>
+                  <token>de</token>
+                  <marker>
+                      <token>part</token>
+                  </marker>
+                  <token postag="D.*" postag_regexp="yes"/>
+              </pattern>
+              <message>Une erreur d’orthographe semble être détectée.</message>
+              <suggestion>par</suggestion>
+              <url>https://www.lalanguefrancaise.com/orthographe/de-par-ou-de-part-orthographe/</url>
+              <example correction="par">Il a réussi de <marker>part</marker> sa pugnacité.</example>
+          </rule>
```

Comment:
The example of the first rule is already found by another rule.
Check out whether your rule is better or worse than that rule:
1) see whether your rule finds all the `<examples>` of that rule
2) `grep -i -w -E "(de part des|[mts](on|a|es)|l(a|es?|'))"` (the `D.*` part of the pattern you'll have to express using regex, this is just an example off the top of my head) in the French corpus and see which rule finds more errors
3) If you see that your rule finds errors the other rule doesn't find, try and "merge" them (e.g. extend the pattern of the old rule or create a new subrule)

Reply:
I checked the rule that already existed. I have created two new rules for this case therefore the rule previously done becomes void. Could it be taken out ?